### ISSUE TITLE
installation issue

### ISSUE THREAD

**Issue by min0609** [Spencerfar/LatentVelo#2]
Hi,
First of all, thank you for this wonderful tool.
I downloaded setup.py and installed latentvelo, but I could not import the module.
<img width="644" alt="image" src="https://user-images.githubusercontent.com/105622235/227128017-6211b84c-fe7d-4f4d-92ae-83375cccd723.png">

I checked the module is installed completely.
<img width="881" alt="image" src="https://user-images.githubusercontent.com/105622235/227128574-68ad96be-a566-4500-ac00-6574fa8f70d4.png">

could you suggest a way to import the module?

Thank you in advance!

**Comment by kxxxjo** [comment]
@Spencerfar, thanks for giving solution to install latentvelo (https://github.com/Spencerfar/LatentVelo/issues/1#issuecomment-1479769582)

After installation according to your suggestion, my kernel was died like below image.

<img width="705" alt="image" src="https://user-images.githubusercontent.com/88973914/228117307-1c2c1be8-df53-4253-8216-4710f290a667.png">

could you let me know how to solve this error and "ModuleNotFoundError"?

Thanks!

Best,
KJ

**Comment by Spencerfar** [comment]
Is there any error while installing?

**Comment by kxxxjo** [comment]
@Spencerfar

No, there was no any error when I install latentvelo using `python setup.py install`

Best,
KJ
